hub.Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /home/gradle/src
 RUN gradle assemble
 
```

Comment:
this file is no longer needed and can be removed. It was used when we were using DockerHub but now we no longer do

Reply:
deleted hub.Dockerfile